### DISCUSSION TITLE
Support for 'nested' blocklist

### DISCUSSION THREAD

**Discussion by eddydc** [AdguardTeam/AdGuardHome#3264] [Ideas]
Hi,
I would like to see support for 'nested' block lists like https://v.firebog.net/hosts/lists.php?type=tick
Currently, one needs to add the content line by line manually. Sure, there are scripts for that, but all they do is mimicking the manual process.
Cheers,
Eddy

**Comment by ameshkov** [comment]
Please take a look at this: https://github.com/AdguardTeam/HostlistCompiler

It does a lot more than just concatenating blocklists.

**Reply by eddydc** [reply]
If I understand it correctly, the configuration should look like
`
{
  "name": "Firebog",
  "sources": [
    {
      "source": "https://v.firebog.net/hosts/lists.php?type=tick"
    }
  ]
}
`
Or didn't I understand it?
